requirements.txt

```diff
@@ -0,0 +1,6 @@
+flywheel-sdk>=12.0.0
+nibabel==3.1.0
+nipype==1.5.0
+numpy==1.18.5
+pydeface==2.0.0
+pydicom==2.0.0
```

Comment:
Nit: I am a fan of using the ~=x.y to allow for upgrading with fixes/patches (as long as package follow semantic versioning)

Reply:
Changed.